src/utils/emptyObjectValues.js

```diff
@@ -0,0 +1,2 @@
+export const emptyObjectValues = obj =>
+  !Object.values(obj).filter(item => item.length).length
```

Comment:
Instead of using `!` on `.length` to produce a boolean value, you could simply use `some` or `every` array methods which return a boolean:
```suggestion
  Object.values(obj).every(item => item.length === 0)
```
Also, according to most conventions, the name of a boolean function (i.e., a function whose return value is of type boolean, either `true` or `false`) should start with words such as `is`, `are`, `can`, `has`, `should`, for example: `isEveryObjectValueEmpty`